lib/src/widgets/switch/ds_switch.widget.dart

```diff
@@ -0,0 +1,85 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+class DSSwitch extends StatelessWidget {
+  /// Creates a customizable switch for use alone or in conjunction with ListTile
+  ///
+  /// Set the active and inactive colors using the parameters [activeColor]
+  /// and [inactiveColor], and use [isActive] to turn the switch on and off.
+  const DSSwitch({
+    Key? key,
+    required this.onChanged,
+    required this.isActive,
+    this.isDisabled = true,
+  }) : super(key: key);
+
+  /// Function callback to use when the [onChanged].
+  final ValueChanged<bool> onChanged;
+
+  /// Enables or disables the widget by changing the color of the switch.
+  final bool isDisabled;
+
+  /// Referring to [isActive] true ou false.
+  final bool isActive;
```

Comment:
At first, I was confused by the difference between **isActive** and **isEnabled** props, due to the semantic similarity of these words. I think that if **isActive** was **isSelected** instead, it'll be more clearer. What you think about this?

Reply:
isActive is an attribute of the switch, isSelected is a native attribute of listTile.which makes the label the same color as the active switch.
I'm going to remove it, because the DS doesn't have this, because it really causes confusion.